src/components/atoms/PublicationStatus/index.tsx

```diff
@@ -36,7 +37,7 @@ const PublicationStatus: React.FC<PublishStatusProps> = ({
       <Text
         size={size === "lg" ? "m" : size === "md" ? "s" : "xs"}
         otherProperties={{ userSelect: "none" }}
-        customColor>
+        color={color}>
```

Comment:
I see that the color prop wasn't being used, so passing it here is fine, but did you make sure this PublicationStatus component being used elsewhere isn't making use of the customColor (aka inheriting a color from a parent component)?